server/http.go

```diff
@@ -295,7 +295,7 @@ func (p *Plugin) handleStartMeeting(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	_, meeting, err := p.postMeeting(user, req.ChannelID, req.Topic)
+	_, meeting, err := p.postMeetingWithDeps(user, req.ChannelID, req.Topic, newClient)
```

Comment:
> In this postMeeting flow both handleStartMeetingWithDeps and authenticateAndFetchUserWithDeps are calling p.GetUserInfo(userID) and p.getOAuthConfig() as well as creating a Client for the same user. This has some performance impact as we are reading KV store twice. I suggest you pass the created client or userInfo to postMeetingWithDeps instead of recreating them.